app/views/users/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>This is the Users#index page.</h1>
+<% @users.each do |user| %>
+  <div class="user_info">
+   <%= image_tag ("#{user.photo}"), alt: "#{user.name}'s photo", width: "100px", height: "80px" %>
+   <span class="text_info">
+   <span class="user_name"><%= "Name: #{user.name}"%></span>
+   <br>
+   <span class="numbers"><%= "Number of posts: #{user.posts_counter}"%></span>
+   </span>
+   <br>
+  </div>
```

Comment:
[OPTIONAL]

- You can use `app/views/users/_user.html.erb` for better practice instead of using `@objects.each`.

```ruby
# app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```

What it will do, it will render each user one by one and display all the user that comes from the controller.

Reply:
Thank you for the suggestion. I will see to it in the future